.circleci/config.yml

```diff
@@ -30,6 +30,17 @@ jobs:
       - run: docker push registry.dev.brainhub.pl/react-carousel/react-carousel:latest
       - run: rancher-compose --project-name react-carousel --verbose up -d --force-upgrade --pull --confirm-upgrade react-carousel
 
+  publish_package:
+    docker:
+      - image: circleci/node:8.11
```

Comment:
does circleci provide newer version of node? LTS version would be the best? 👍

Reply:
@damian-brainhub 

in CircleCI I was using only Node.js 8 because I prefer an LTS